### PR TITLE
Added 米 (cross and star) style guide.

### PR DESCRIPTION
**Pull request by derpferd**

### CODE REVIEW

File: frontend/index.html [hunk 1]
```diff
@@ -41,6 +41,7 @@
     <option value="0">None</option>
     <option value="1">X Star</option>
     <option value="2" selected>+ Cross</option>
+    <option value="3" selected>米 Cross &amp; Star</option>
```
Comment (lucivpav) [inline frontend/index.html]:
```suggestion
    <option value="3">米 Cross &amp; Star</option>
```
Let's make only one option selected by default. I suggest we keep the `cross` as the default option, as users are used to it.

Comment (derpferd) [reply frontend/index.html]: Done.

Comment (derpferd) [reply frontend/index.html]: Done

**Comment by lucivpav** [comment]
Thank you for the PR!

**Pull request merged**